security_center/incident_handling_process.rst

```diff
@@ -4,14 +4,19 @@ Security Incident Handling Process
 Reporting
 ---------
 
-If you think you have found a security vulnerability, then please send an email
-to the Trusted Firmware security team at security@lists.trustedfirmware.org.
-You may also use the project specific security team aliases
-:doc:`here <mailing_aliases>`. These are private teams of security officers
-who will help verify the security vulnerability, develop and release a fix,
-and disclose the vulnerability details responsibly. The security teams are
-referred to from now on as "us/we/our". We do our best to respond and fix any
-issues as soon as possible.
+If you think you have found a security vulnerability in Trusted Firmware
+infrastructure, then please follow the
+`Linaro security policy <https://www.linaro.org/.well-known/security.txt>`__
```

Comment:
Should we also / rather refer to https://www.linaro.org/vdp?

Reply:
OK. I was concerned about the vdp link changing but using both works!

Reply:
I don't think we have any plans of changing that and if we would, it's still quick and easy to update the link in the docs.